steps/gcp-step-cloud-run-deploy/spec.schema.json

```diff
@@ -0,0 +1,131 @@
+
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "credentials": {
```

Comment:
I don't believe this was ever updated to use the GCP connection type. We can do that now, or you can use the old default, which is that it should be simply:

```json
{
  "credentials": {
    "type": "object",
    "properties": {
      "service-account.json": {
        "type": "string"
      }
    }
  }
}
```